### PR TITLE
Fixes size rounding issues and alters how the graph backend stores no…

### PR DESCRIPTION
**Pull request by jlaura**
…des and edges.

### CODE REVIEW

**Comment by jlaura** [comment]
@all Give this a look and also let me know if this does not fix the issue with edge ordering.

File: autocnet/vis/graph_view.py [hunk 1]
```diff
@@ -211,119 +207,5 @@ def plot_edge(edge, ax=None, clean_keys=[], image_space=100,
...
-            # transformation from data coordinates to display coordinates
-            plt.plot([keypointImage1[0], keypointImage2[0]],
-                     [keypointImage1[1], keypointImage2[1]],
-                     color='g',
-                     marker='o',
-                     markeredgecolor='g',
-                     markersize=featurePointSize,
-                     linewidth=lineWidth,
-                     alpha=0.5)
-    return plt.imshow(displayBox, cmap='Greys')
```
Comment (Kelvinrr) [inline autocnet/vis/graph_view.py]: I am sure there is a good reason, but why delete all of this? Simply not needed anymore? Is it related to the original problem?

File: autocnet/graph/edge.py [hunk 2]
```diff
@@ -113,15 +113,15 @@ def compute_fundamental_matrix(self, clean_keys=[], **kwargs):
 
         all_source_keypoints = self.source.get_keypoint_coordinates(matches['source_idx'])
         all_destin_keypoints = self.destination.get_keypoint_coordinates(matches['destination_idx'])
-
+        print(matches)
```
Comment (kree) [inline autocnet/graph/edge.py]: pull out debugging print statement

File: autocnet/graph/edge.py [hunk 3]
```diff
@@ -113,15 +113,15 @@ def compute_fundamental_matrix(self, clean_keys=[], **kwargs):
...
         matches, mask = self._clean(clean_keys)
-
+        print(matches, mask.sum())
```
Comment (kree) [inline autocnet/graph/edge.py]: pull out debugging print statement

File: autocnet/graph/edge.py [hunk 4]
```diff
@@ -113,15 +113,15 @@ def compute_fundamental_matrix(self, clean_keys=[], **kwargs):
...
         s_keypoints = self.source.get_keypoint_coordinates(matches['source_idx']).values
         d_keypoints = self.destination.get_keypoint_coordinates(matches['destination_idx']).values
         transformation_matrix, fundam_mask = od.compute_fundamental_matrix(s_keypoints,
                                                                            d_keypoints,
                                                                            **kwargs)
-
+        print(fundam_mask)
```
Comment (kree) [inline autocnet/graph/edge.py]: pull out debugging print statement

File: autocnet/graph/edge.py [hunk 5]
```diff
@@ -139,6 +139,7 @@ def compute_fundamental_matrix(self, clean_keys=[], **kwargs):
 
         # Set the initial state of the fundamental mask in the masks
         self.masks = ('fundamental', mask)
+        print(self.masks)
```
Comment (kree) [inline autocnet/graph/edge.py]: pull out debugging print statement

File: autocnet/vis/graph_view.py (hunk 1 above)
Comment (jlaura) [inline autocnet/vis/graph_view.py]: Not needed and always available in the git history.

File: autocnet/graph/network.py [hunk 6]
```diff
@@ -656,10 +664,34 @@ def create_edge_subgraph(self, edges):
                 H.adj[v] = H.adjlist_dict_factory()
             # Copy the edge attributes.
             H.edge[u][v] = self.edge[u][v]
-            H.edge[v][u] = self.edge[v][u]
+            #H.edge[v][u] = self.edge[v][u]
         H.graph = self.graph
+        print(sum(H.degree(weight=None).values()) / 2)
```
Comment (kree) [inline autocnet/graph/network.py]: pull out debugging print statement

**Comment by jlaura** [comment]
I was on a crazy print roll I guess!

**Comment by jlaura** [comment]
@all should be good to go after the CI does its thing.

File: autocnet/matcher/outlier_detector.py [hunk 7]
```diff
@@ -316,9 +316,8 @@ def mirroring_test(matches):
                  otherwise, they will be false. Keypoints with only one match will be False. Removes
                  duplicate rows.
     """
-    duplicate_mask = matches.duplicated(subset=['source_idx', 'destination_idx', 'distance'],
-                                    keep='last')
-
+    import time
```
Comment (kree) [inline autocnet/matcher/outlier_detector.py]: Are you using this?

**Comment by kree** [comment]
@jlaura I added one last quick comment.

**Comment by jlaura** [comment]
@kree pulled it.  Once passed, anyone can merge.

**Pull request merged**